fastlane/Fastfile

```diff
@@ -29,4 +29,32 @@ desc "Run unit tests"
       	clean: true
       )
   end
+  
+desc "Sonar Cloud Scanner"
+  lane :metrics do
+
+    scan(scheme: "PaystackiOS Tests",
+      code_coverage: true,
+      output_directory: "./sonar-reports",
+      devices: "iPhone 8")
+     
+    slather(cobertura_xml: true,
+      scheme: "PaystackiOS",
+      proj: "./Paystack.xcodeproj",
+      output_directory: "./sonar-reports",
+      workspace: "./Paystack.xcworkspace")
+
+     swiftlint(output_file: "./sonar-reports/swiftlint.txt",
```

Comment:
Once we merge this and start running this as part of our primary workflow it would be better to remove swiftlint from running as part of danger. Or else it would be repeated

Reply:
Agreed, if we want to setup custom rules here we can do that with swiftlint.yml and set it in the fastlane plugin.